_content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Skt. dāyāda- 'Eating Away at the Inherited/Entrusted': The Transformation of Inherited Indo-European Phraseology in the Buddhist Legend of Ajātaśatru"
+authors:
+  - "Olav Hackstein"
+external_url: "https://poj.peeters-leuven.be/content.php?url=article&id=3134549&journal_code=JIABS&download=yes"
+drive_links:
+  - "https://drive.google.com/file/d/1NVzUGptMFQaElYOfbL64yDfZb5p90DSj/view?usp=sharing"
+course: karma
+tags:
+  - language
```

Comment:
let's maybe also add the `pali-dictionaries` tag? Not exactly right, but I don't yet have a Pali Philology tag, so the lexicographic tag will have to do double duty for now...

And possibly also the `historiography` and `mythology` tags, as it's such a fun example (to me at least!)